packages/dds/tree/src/test/shared-tree/treeCheckout.spec.ts

```diff
@@ -142,6 +142,62 @@ describe("sharedTreeView", () => {
 				"editStart",
 			]);
 		});
+
+		describe("commitApplied", () => {
+			it("is fired for data and schema changes", () => {
+				const provider = new TestTreeProviderLite(1);
+				const checkout1 = provider.trees[0].checkout;
```

Comment:
apologies if nitpicky but can we just call this `checkout`? `checkout1` implies to me that there is a `checkout2` somewhere - but I don't see one. But, it's fine if this is an intentional convention that you prefer.

Reply:
No, this was just a case of copy-and-trim. I agree `checkout` is best.